Strategy11/business-directory-plugin
# Fix Term Title, Astra Theme Output, and Region Title Issues

### Issue Link:
https://github.com/Strategy11/business-directory-premium/issues/141

This PR addresses three distinct issues related to term titles, the Astra theme, and region titles in block themes:

## 1. Term Title Issue:
The term title in block themes has been fixed to properly display the taxonomy term name when the block is part of a query loop.

### How to test:
1. Activate the 2022 theme (or any other block theme affected by this issue).
2. Navigate to a category or tag page.
3. Confirm that the page title displays the correct term title instead of the listing name.

### WP 2022 Theme - Before/After Screenshots:
#### Before
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/69119241/229562401-5b8337eb-087c-4e93-924a-f7f4eb848cb6.png">

#### After
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/69119241/229562518-aa7cbd3a-ac5b-4bbe-82e4-7453867400f6.png">

## 2. Astra Theme Output Issue:
An issue causing incorrect term output in the Astra theme has been resolved.

### How to test:
1. Activate the Astra theme.
2. Navigate to a category or tag page.
3. Confirm that the page output displays the correct term output.

### Astra Theme - Before/After Screenshots:
#### Before
![image](https://user-images.githubusercontent.com/69119241/230098004-6d191c27-5889-4f81-b94c-11e5e956b733.png)

#### After
![image](https://user-images.githubusercontent.com/69119241/230098050-a2016688-ae5f-4ffc-9f80-5a7113c295ec.png)

## 3. Region Title Issue:
A problem affecting the display of region titles has been corrected.


### How to test:
1. Activate the 2022 theme (or any other block theme affected by this issue).
2. Navigate to a region page.
3. Confirm that the page title displays the correct term title instead of the listing name.

### WP 2022 Theme - Before/After Screenshots:
#### Before
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/69119241/230099258-498ec610-1162-4d95-acda-3e6b0fdadb42.png">

#### After
<img width="1504" alt="image" src="https://user-images.githubusercontent.com/69119241/230099197-9af996cf-9646-4d53-a6b2-94a0e01252d5.png">
